resources/task.rb

```diff
@@ -0,0 +1,149 @@
+property :task_name, String, desired_state: false, identity: true, name_attribute: true
+property :task_source, String, desired_state: false, identity: true, default: ''.freeze
+property :task_crontab, String, desired_state: false, identity: true, default: '0 1 * * * ?'.freeze
```

Comment:
This is a "desired_state" property and not an "identity" one

Reply:
done